.github/workflows/regression-tests.yml

```diff
@@ -0,0 +1,110 @@
+name: Regression Tests
+'on':
+  workflow_dispatch:
+    inputs:
```

Comment:
It might be great to also add the zk-prover and the zkevm-contracts to this list in a future ticket.

Reply:
spoke with john about prover piece, intentionally not adding as very cumbersome and long build

contracts version is a good idea for future version!

Reply:
follow up ticket: [DVT-1456](https://polygon.atlassian.net/browse/DVT-1456)